tests/backup/backup_resiliency_test.go

```diff
@@ -1198,3 +1199,235 @@ var _ = Describe("{DeleteIncrementalBackupsAndRecreateNew}", func() {
+					getGlobalBucketName(provider), orgID, "")
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating backup location %s", bkpLocationName))
+			}
+		})
+		Step("Registering source and destination clusters for backup", func() {
+			log.InfoD("Registering source and destination clusters for backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Getting admin context")
+			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Creating source cluster %s and destination cluster %s", SourceClusterName, destinationClusterName))
+			srcClusterStatus, _ = Inst().Backup.RegisterBackupCluster(orgID, SourceClusterName, "")
+			dash.VerifyFatal(srcClusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying source cluster %s status", SourceClusterName))
+			destClusterStatus, destClusterUid = Inst().Backup.RegisterBackupCluster(orgID, destinationClusterName, "")
+			dash.VerifyFatal(destClusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying destination cluster %s status", destinationClusterName))
+		})
+		Step("Taking backup of applications", func() {
+			log.InfoD("Taking backup of applications")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Getting admin context")
+			for _, namespace := range appNamespaces {
+				sem <- struct{}{}
+				backupName := fmt.Sprintf("%s-%s", BackupNamePrefix, namespace)
+				backupNames = append(backupNames, backupName)
+				wg.Add(1)
+				go func(backupName string, namespace string) {
+					defer GinkgoRecover()
+					defer wg.Done()
+					defer func() { <-sem }()
+					// Here we are using destination cluster as application cluster
+					_, err = CreateBackupWithoutCheck(backupName, destinationClusterName, bkpLocationName, backupLocationUID,
+						[]string{namespace}, labelSelectors, orgID, destClusterUid, "", "", "", "", ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Taking backup %s of application- %s", backupName, namespace))
+				}(backupName, namespace)
+			}
+			wg.Wait()
+			log.Infof("The list of backups taken are: %v", backupNames)
+		})
+		Step("Reboot one worker node on application cluster when backup is in progress", func() {
+			log.InfoD("Switching cluster context to application[destination] cluster")
+			SetDestinationKubeConfig()
+			listOfWorkerNodes = node.GetWorkerNodes()
+			err := Inst().N.RebootNode(listOfWorkerNodes[0], node.RebootNodeOpts{
+				Force: true,
+				ConnectionOpts: node.ConnectionOpts{
+					Timeout:         rebootNodeTimeout,
+					TimeBeforeRetry: rebootNodeTimeBeforeRetry,
+				},
+			})
+			dash.VerifySafely(err, nil, fmt.Sprintf("Rebooting worker node %v", listOfWorkerNodes[0].Name))
+			log.InfoD("Switching cluster context back to source cluster")
+			err = SetSourceKubeConfig()
+			dash.VerifyFatal(err, nil, "Switching back context to source cluster")
```

Comment:
Use `log.FailOnError` for error check everywhere

Reply:
@mkoppal-px log.FailOnError and dash.VerifyFatal does the same work
Wouldn't it be redundant?
log.FailOnError will execute only in case of failure, but dash.VerifyFatal will print what we are verifying irrespective of pass or fail

Reply:
I meant replace VerifyFatal with log.FailOnError. FailOnError will also print it

Reply:
done